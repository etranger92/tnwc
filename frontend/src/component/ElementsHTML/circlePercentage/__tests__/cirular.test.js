import * as React from 'react';
import CirclePercent from "../circular.tsx"
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the innerHTML of the circulePercent", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CirclePercent />);
  });
  test("Snapshot circulePercent ", () => {
    //It works
 expect(toJson(wrapper)).toMatchSnapshot();

});
});
