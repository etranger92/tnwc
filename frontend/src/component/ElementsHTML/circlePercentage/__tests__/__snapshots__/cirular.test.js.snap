// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the innerHTML of the circulePercent Snapshot circulePercent  1`] = `
<AnimatedProgressProvider
  duration={2}
  easingFunction={[Function]}
  repeat={false}
  valueEnd={0}
  valueStart={0}
>
  <Component />
</AnimatedProgressProvider>
`;
